functions.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Enqueues the design system CSS stylesheet on the frontend of the WordPress site.
+ *
+ * This function uses the `wp_enqueue_style` function to load the `index.css` file from the `dist/styles` directory
+ * within the current theme directory.
+ *
+ * @since 1.3.0
+ */
+function enqueue_global_styles() {
```

Comment:
I think this should be prefixed or it could collide with other global-scope functions named the same thing. Something like `dswt_enqueue_global_styles()` maybe.